internal/page/donor/check_your_lpa.go

```diff
@@ -13,22 +15,102 @@ import (
 )
 
 type checkYourLpaData struct {
-	App       page.AppData
-	Errors    validation.List
-	Lpa       *page.Lpa
-	Form      *checkYourLpaForm
-	Completed bool
+	App         page.AppData
+	Errors      validation.List
+	Lpa         *page.Lpa
+	Form        *checkYourLpaForm
+	Completed   bool
+	CanContinue bool
 }
 
-func CheckYourLpa(tmpl template.Template, donorStore DonorStore, shareCodeSender ShareCodeSender, notifyClient NotifyClient, certificateProviderStore CertificateProviderStore) Handler {
+type checkYourLpaNotifier struct {
```

Comment:
I pulled this thing out because the code here was getting very nested

Reply:
Much easier to parse now